sample/src/main/java/br/com/stant/libraries/card_show_taken_pictures_view/MainActivity.java

```diff
@@ -31,28 +31,35 @@ protected void onCreate(Bundle savedInstanceState) {
 
         mBinding.cardShowViewTakenPicturesView.setActivity(this);
         mBinding.cardShowViewTakenPicturesView.setExampleImages();
-        mBinding.cardShowViewTakenPicturesView.setOnSavedCardListener(new CardShowTakenPictureViewContract.OnSavedCardListener() {
-            @Override
-            public void onSaved(List<CardShowTakenImage> imagesAsAdded, List<CardShowTakenImage> imagesAsRemoved) {
+        mBinding.cardShowViewTakenPicturesView.setOnSavedCardListener(
+                new CardShowTakenPictureViewContract.OnSavedCardListener() {
+                    @Override
+                    public void onSaved(List<CardShowTakenImage> currentImages,
+                                        List<CardShowTakenImage> imagesAsAdded,
+                                        List<CardShowTakenImage> imagesAsRemoved) {
 
-            }
+                    }
 
-            @Override
-            public void onCancel() {
+                    @Override
+                    public void onCancel() {
 
-            }
-        });
+                    }
+                }
+        );
 
-        if(gallery != null)
+        if (gallery != null)
             mBinding.cardShowViewTakenPicturesView.setCardImages(gallery.getImages());
 
-        mBinding.cardShowViewTakenPicturesView.setBinding(mBinding.cardShowViewTakenPicturesView,"Denis Vieira", new Date());
+        mBinding.cardShowViewTakenPicturesView.setBinding(mBinding.cardShowViewTakenPicturesView,
+                "Denis Vieira", new Date());
 //        mBinding.cardShowViewTakenPicturesView.setImagesQuantityLimit(15, null);
         mBinding.cardShowViewTakenPicturesView.setIsMultipleGallerySelection(true);
-        mBinding.cardShowViewTakenPicturesView.setStrokeColor(ContextCompat.getColor(getApplicationContext(), R.color.colorPrimary));
-        mBinding.cardShowViewTakenPicturesView.setBackgroundColor(ContextCompat.getColor(getApplicationContext(), R.color.colorPrimary));
-        mBinding.cardShowViewTakenPicturesView.enableSaveOnlyMode(ContextCompat.getDrawable(getBaseContext(), R.drawable.ic_cam), "oii",
-                ContextCompat.getDrawable(getBaseContext(), R.drawable.ic_close_dialog), "Tchau");
+        mBinding.cardShowViewTakenPicturesView.setStrokeColor(ContextCompat.getColor(getApplicationContext(),
```

Comment:
On this class, i believe that we create methods to test contexts, for not get very free . Create a method with your context and call here .